rackcity/models/asset.py

```diff
@@ -32,10 +33,32 @@ def validate_owner(value):
         )
 
 
-class AbstractAsset(models.Model):
+class AbstractID(models.Model):
     id = models.AutoField(primary_key=True)
 
 
+class AbstractAsset(AbstractID):
```

Comment:
I would name these differently - specifically I think AbstractID is unclear bc it doesn't have the word Asset

Reply:
changing to AssetID